src/ar.erl

```diff
@@ -118,7 +118,8 @@ main("") ->
 			{"enable (feature)", "Enable a specific (normally disabled) feature. For example, subfield_queries."},
 			{"disable (feature)", "Disable a specific (normally enabled) feature. For example, api_compat mode."},
 			{"gateway (port) (domain)", "Run a gateway on the specified port and domain"},
-			{"custom_domain (domain)", "Add a domain to the list of supported custom domains."}
+			{"custom_domain (domain)", "Add a domain to the list of supported custom domains."},
+			{"accept_requests_per_minute_limit (number)", "Limit the number of accepted requests per minute per IP. Default is 2000."}
```

Comment:
I'm suggesting calling this for "requests_per_minute_limit" instead and changing the description to:

Limit the number of HTTP requests allowed per IP address per minute. Default is 2000.

Reply:
I wanted to distinguish it this way from the outbound requests, but the context is hopefully clear enough so I think it's fine to go with your suggestion.